packages/balances-react/package.json

```diff
@@ -36,28 +36,28 @@
     "@talismn/util": "workspace:*",
     "anylogger": "^1.0.11",
     "blueimp-md5": "2.19.0",
-    "dexie": "^3.2.4",
+    "dexie": "^4.0.9",
     "dexie-react-hooks": "^1.1.7",
-    "jotai": "2.x",
-    "jotai-effect": "0.x",
+    "jotai": "~2.10.1",
+    "jotai-effect": "~1.0.3",
     "lodash": "4.17.21",
-    "react-use": "^17.4.0",
+    "react-use": "^17.5.1",
     "rxjs": "^7.8.1"
   },
   "devDependencies": {
     "@talismn/eslint-config": "workspace:*",
     "@talismn/tsconfig": "workspace:*",
-    "@types/jest": "^27.5.1",
-    "@types/lodash": "^4.14.180",
-    "@types/react": "^18.0.17",
-    "eslint": "^8.4.0",
+    "@types/jest": "^29.5.14",
+    "@types/lodash": "^4.17.12",
+    "@types/react": "^18.3.12",
+    "eslint": "^8.57.1",
     "jest": "^29.7.0",
-    "react": "^18.2.0",
-    "ts-jest": "^29.1.1",
-    "typescript": "^5.2.2"
+    "react": "^18.3.1",
+    "ts-jest": "^29.2.5",
+    "typescript": "^5.6.3"
   },
   "peerDependencies": {
-    "@polkadot/util-crypto": "12.x",
+    "@polkadot/util-crypto": "13.2.1",
```

Comment:
I think we want peerDependencies to remain as `13.x` so we use whatever version is installed on the dapp - rather than pulling in an additional copy of util-crypto just for this library